src/main/CoverageTrack.js

```diff
@@ -35,10 +35,10 @@ function extractSummaryStatistics(reads: Array<SamRead>, contig: string) {
   });
   var maxCoverage = _.max(binCounts);
 
-  binCounts = _.map(binCounts, (count, position) => ({position: Number(position), count}));
-  binCounts = _.sortBy(binCounts, bin => bin.position);
+  var posCounts = _.map(binCounts, (count, position) => ({position: Number(position), count}));
```

Comment:
FYI FlowType was having trouble tracking the changing type of this single variable.